data/keywords.yml

```diff
@@ -0,0 +1,248 @@
+- title: JSON Schema Keywords
+  description: >-
+    JSON Schema keywords are the building blocks of JSON Schema. They are used
+    to define the structure of a JSON document
+  categorys:
```

Comment:
We need to change the structure to store URLs and titles of the URLs and also allow 1 keyword to belong to multiple vocabularies without having to duplicate it. Find below an example :

````
- name: $anchor
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/learn/json-schema-examples#ecommerce-system'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/structuring#anchor'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/anchor'
      title:
- name: $comment
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/understanding-json-schema/reference/comments#comments'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/comment'
      title:
- name: $defs
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/learn/miscellaneous-examples'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/structuring#defs'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/defs'
      title:
- name: $dynamicAnchor
  vocabulary: [core]
  links:
    - url: 'https://www.learnjsonschema.com/2020-12/core/dynamicanchor'
      title:
- name: $dynamicRef
  vocabulary: [core]
  links:
    - url: 'https://www.learnjsonschema.com/2020-12/core/dynamicref'
      title:
- name: $id
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/basics#declaring-a-unique-identifier'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/id'
      title:
- name: $ref
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/learn/getting-started-step-by-step#create-a-schema-definition'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/basics#declaring-a-json-schema'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/reference/array#unevaluateditems'
      title:
    - url: 'https://json-schema.org/understanding-json-schema/structuring#id'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/ref'
      title:
- name: $schema
  vocabulary: [core]
  links:
    - url: 'https://json-schema.org/understanding-json-schema/reference/schema'
      title:
    - url: 'https://www.learnjsonschema.com/2020-12/core/schema'
      title:
````